lib/service-worker-builder.js

```diff
@@ -85,7 +85,13 @@ module.exports = class ServiceWorkerBuilder {
 
   _rollupTree(tree, entryFile, destFile) {
     let rollupReplaceConfig = {
-      include: ['/**/ember-service-worker/**/*.js'],
+      include: [
+        '/**/ember-service-worker/**/*.js',
+        // same as above but for windows paths like D:\my-ember\ember-service-worker\whatever\sw.js;
+        // the original one doesn't work when the source code and the %TMP% folder, which ember started to use after v3.5, are located on different logical drives
+        // which is quite common in Windows.
+        /^[a-z]:\/(?:[^\/:*?"<>|\r\n]+\/)*ember-service-worker\/(?:[^\/:*?"<>|\r\n]+\/)*[^\/:*?"<>|\r\n]*\.js$/i
```

Comment:
It appears that you are checking for "/" in the regex rather than "\" in the comment. Which one is the intended? I seem to recall windows paths being more like your example but it's been a while....

Reply:
Yes, it is true. Windows paths do have `\` as path separators, but somewhere in the code all the paths are "normalized" to use only `/`s. I may fix the comment though. What would you say?

Reply:
Sorry for the silence... holidays and all. I'm trying to find where that happens but I'm coming up short. I would just make the comment as explicit as it needs to be so that the next person looking at this understands why the regex is the way it is. Maybe `same as above but for windows paths like D:\my-ember\ember-service-worker\whatever\sw.js; which are updated before this step to D:/my-ember/ember-service/worker/whatever/sw.js`

Reply:
@eshtadc done!